scripts/auto-rebase/rebase.sh

```diff
@@ -43,6 +42,159 @@ title() {
     echo -e "\E[34m$1\E[00m";
 }
 
+# LVMS is not integrated into the ocp release image, so the work flow does not fit with core component rebase.  LVMS'
+# operator bundle is the authoritative source for manifest and image digests.
+download_lvms_operator_bundle(){
+    title 'downloading LVMS operator bundles'
+    declare -A images=(["amd64"]="${1}" ["arm64"]="${2:-}")
+    local LVMS_STAGING="${STAGING_DIR}/lvms"
+    for arch in "${!images[@]}"; do
+        # TODO special case arm64 builds of topolvm since they are not available in 4.12
+        if [[ "$arch" == "arm64" && "${images[${arch}]}" == "" ]]; then
+            continue
+        fi
+        img="${images[$arch]}"
+        mkdir -p "$LVMS_STAGING/$arch"
+        pushd "$LVMS_STAGING/$arch" || return 1
+        oc image extract \
+            --registry-config "$PULL_SECRET_FILE" \
+            --path /manifests/:. "$img" \
+            --filter-by-os "$arch" \
+            ||  {
+                    popd
+                    return 1
+                }
+        popd || return 1
+      done
+}
+
+parse_images() {
+    local src=$1
+    yq '.spec.relatedImages[]? | [.name, .image] | @csv' $src > "$(dirname "$src")/images"
```

Comment:
Can we also pass the output file as an argument, so that it's not defined in two places?

Reply:
+1